### PR TITLE
numpy - arcsinh

### PR DESCRIPTION
**Pull request by Darshan2104**
Numpy - arcsinh #3004

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py [hunk 1]
```diff
@@ -56,3 +56,52 @@ def test_numpy_sinh(
         subok=True,
         test_values=False,
     )
+
+# arcsinh
```
Comment (Unevilicorn) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py:60]: There is a lint issue for this function, please fix this

File: ivy/functional/frontends/numpy/mathematical_functions/hyperbolic_functions.py [hunk 2]
```diff
@@ -24,3 +24,25 @@ def sinh(
 
 
 sinh.unsupported_dtypes = {"torch": ("float16",)}
+
+@from_zero_dim_arrays_to_float
```
Comment (Unevilicorn) [inline ivy/functional/frontends/numpy/mathematical_functions/hyperbolic_functions.py:28]: lint issue in this function

**Review by Unevilicorn — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py (hunk 1 above)
Comment (Darshan2104) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py:62]: Hey, @Unevilicorn can you guide me on how can I solve this issue? I'm new to this.
Thanks

**Comment by Unevilicorn** [comment]
This function is now completed from what I can see in the original todo list.

**Pull request closed without merging**